### PR TITLE
Rename Dodge The Creeps files to snake case

### PR DESCRIPTION
**Pull request by russellsanborn**
This follows the style-guide:

* https://docs.godotengine.org/en/stable/tutorials/best_practices/project_organization.html#style-guide

For reference, a documentation PR exists to update references to the demo.

* https://github.com/godotengine/godot-docs/pull/9562

### CODE REVIEW

File: 2d/dodge_the_creeps/main.tscn [hunk 1]
```diff
@@ -1,11 +1,11 @@
 [gd_scene load_steps=8 format=3 uid="uid://cyfwty2q3rdse"]
 
-[ext_resource type="Script" path="res://Main.gd" id="1"]
-[ext_resource type="PackedScene" uid="uid://rkdnhqgf2hpj" path="res://Mob.tscn" id="2"]
-[ext_resource type="PackedScene" uid="uid://4vwrqjegqwpj" path="res://Player.tscn" id="3"]
-[ext_resource type="PackedScene" uid="uid://ccqoreueuxdb7" path="res://HUD.tscn" id="4"]
-[ext_resource type="AudioStream" uid="uid://q2pf4fr8d0ks" path="res://art/House In a Forest Loop.ogg" id="5"]
-[ext_resource type="AudioStream" uid="uid://dw26fpygeag8o" path="res://art/gameover.wav" id="6"]
+[ext_resource type="Script" path="res://main.gd" id="1_0r6n5"]
```
Comment (russellsanborn) [inline 2d/dodge_the_creeps/main.tscn:3]: When i renamed the files within Godot, these id properties added suffixes, but that did not happen in the other `.tscn` files. I am new here, so i don't know if it is expected behavior.

Comment (AThousandShips) [reply 2d/dodge_the_creeps/main.tscn:3]: That's because the IDs were in 3.x format, but only updated when saving some dependency probably

**Review by aaronfranke — APPROVED** [review]
Assuming this runs, and assuming the docs PR is merged in parallel, this looks good to me.

**Review by skyace65 — APPROVED** [review]

**Review by Calinou — APPROVED** [review]
Thanks!

**Pull request merged**